sdk/storage/src/core/clients/storage_client.rs

```diff
@@ -337,6 +337,42 @@ impl StorageClient {
         }
     }
 
+    #[cfg(feature = "mock_transport_framework")]
+    /// Create a new instance of `CosmosClient` using a mock backend. The
+    /// transaction name is used to look up which files to read to validate the
+    /// request and mock the response.
+    // TODO(yosh): consider adding a general way to replace transports, and remove this method.
```

Comment:
You can remove this (it refers to Cosmos anyway) - when we change how the mock stuff is initialized, I'll make sure this gets changed.